package.json

```diff
@@ -64,7 +65,6 @@
     "devDependencies": {
         "prettier": "2.0.5",
         "eslint-config-prettier": "^2.9.0",
-        "eslint-plugin-prettier": "^2.6.0",
-        "react-confirm-alert": "^3.0.5"
```

Comment:
Isn't it useful ?

Reply:
Totally useless. this dependency has been abandoned